src/FCharacter.cpp

```diff
@@ -41,6 +41,9 @@ void FCharacter::move(FDirection direction)
 	switch (direction)
 	{
 		case up:
+			//TODO Add ladders
+			jump();
```

Comment:
I like it when things are this easy, just add a call to `jump()` and stuff works :D @dekespo